patches/minecraft/net/minecraft/client/audio/SoundHandler.java.patch

```diff
@@ -0,0 +1,40 @@
+--- ../src-base/minecraft/net/minecraft/client/audio/SoundHandler.java
++++ ../src-work/minecraft/net/minecraft/client/audio/SoundHandler.java
+@@ -67,6 +67,7 @@
+     {
+         this.field_147697_e.func_148763_c();
+ 
++        Map<ResourceLocation, SoundList> resources = new java.util.IdentityHashMap<>();
+         for (String s : p_110549_1_.func_135055_a())
+         {
+             try
+@@ -79,6 +80,8 @@
+ 
+                         for (Entry<String, SoundList> entry : map.entrySet())
+                         {
++                            resources.put(new ResourceLocation(s, entry.getKey()), entry.getValue());
++                            if (false)
+                             this.func_147693_a(new ResourceLocation(s, entry.getKey()), entry.getValue());
```

Comment:
You could just remove this line instead of dead-coding it. Making dead code in patches is important if its  a big chunk. Single line removals are quite simple patch wise.